server/test/com/cloud/ha/HighAvailabilityManagerImplTest.java

```diff
@@ -113,12 +110,18 @@
     public void setup() throws IllegalArgumentException,
             IllegalAccessException, NoSuchFieldException, SecurityException {
         highAvailabilityManager = new HighAvailabilityManagerImpl();
-        for (Field injectField : HighAvailabilityManagerImpl.class
-                .getDeclaredFields()) {
+        for (Field injectField : HighAvailabilityManagerImpl.class.getDeclaredFields()) {
             if (injectField.isAnnotationPresent(Inject.class)) {
                 injectField.setAccessible(true);
-                injectField.set(highAvailabilityManager, this.getClass()
-                        .getDeclaredField(injectField.getName()).get(this));
+                injectField.set(highAvailabilityManager, this.getClass().getDeclaredField(injectField.getName()).get(this));
+            } else if (injectField.getName().equals("_workers")) {
```

Comment:
Yes. "_workers" is the set of worker threads that perform HA related tasks in HighAvaiabilityManagerImpl class. As part of the test wakeupWorkers() methods get invoked which needs _workers to be initialized.